en/scripting/create-destroy.md

```diff
@@ -108,6 +108,6 @@ cc.Class({
 
 ### destroy and removeFromParent
 
-After calling a node's `removeFromParent`, in-memory data is not guaranteed to be released because there may be some logical problems that cause the program to still refer to the object. So if a node is no longer used, please call its `destroy` instead of `removeFromParent`. `destroy` will not only activate the components `onDestroy`, but also reduce the probability of memory leaks, while reducing the consequences of memory leak.
+After calling `removeFromParent` of a node, in-memory data is not released because there is some logical problem that causes the program to still refer to the object. **So if a node is no longer used, please call its `destroy` directly instead of `removeFromParent`, otherwise a memory leak will result.**
```

Comment:
```suggestion
After calling `removeFromParent` interface on a node, the node is not released from memory because the engine still holds its data internally. **If a node is no longer used, please call its `destroy` directly instead of `removeFromParent`, otherwise, a memory leak will result.**
```